packages/graphile-build-pg/src/plugins/PgIntrospectionPlugin.js

```diff
@@ -500,7 +592,8 @@ export default (async function PgIntrospectionPlugin(
     const handleChange = throttle(
       async () => {
         debug(`Schema change detected: re-inspecting schema...`);
-        introspectionResultsByKind = await introspect();
+        //TODO pretty sure that introspection here is unecessary if we are just calling rebuild immediately after
+        //introspectionResultsByKind = await introspect();
```

Comment:
But... this function handles changes to the schema. If it doesn't re-introspect then the schema will be rebuilt without any changes!